pages/roles.js.php

```diff
@@ -257,7 +257,7 @@ function(data) {
                     $('.infotip').tooltip();
 
                     // Adapt select
-                    $('#folders-depth').val('').change();
+                    $('#folders-depth').val('').empty();
```

Comment:
`empty()` should instead replace `val('')` while retaining `change()`

```javascript
$('#folders-depth').empty().change();
```

`empty()` will clear all old options and `change()` optionally trigger event listeners.